monkey/monkey_island/cc/services/log_service/setup_log_service.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+from flask_restful import Api
+
+from common import DIContainer
+from monkey_island.cc.event_queue import IIslandEventQueue, IslandEventTopic
+from monkey_island.cc.server_utils.island_logger import get_log_file_path
+
+from .file_agent_log_repository import FileAgentLogRepository
+from .flask_resources.register_resources import register_resources
+from .i_agent_log_repository import IAgentLogRepository
+
+
+def setup_log_service(api: Api, container: DIContainer, data_dir: Path):
+    island_event_queue = container.resolve(IIslandEventQueue)
+    repository = container.resolve(FileAgentLogRepository)
+    container.register_instance(IAgentLogRepository, repository)
```

Comment:
Why are we registering this with the DI container? Nothing outside this service should need it.

Reply:
Whoops, leftover from earlier